lib/routes.js

```diff
@@ -32,9 +32,23 @@ module.exports = function(self, options) {
           status: (typeof (err) === 'string') ? err : 'error'
         });
       }
-      return res.send(_.assign({
-        status: 'ok'
-      }, results));
+
+      // On success, grab current version of reverted doc and include url for client-side redirect
+      return self.apos.docs.db.find(req, { _id: id }).toArray(function(err, docs) {
```

Comment:
You are using the mongodb collection but passing a req, that doesn't make sense. You want `docs.find` not `docs.db.find`. Pretty sure it can't work as written.